### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by kurtbahartr**
#### Describe your PR, what does it fix/add?
1. Adds an opt-in role for VMware Workstation.
2. Adds a role for internationalization.
2.1. Installs fcitx5 and its dependencies.
2.2. Installs additional fonts and/or IMEs for Chinese, Japanese, Korean and/or toki pona when enabled.
3. Fixes autocompletion for `ls`.

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
N/A

#### Is it ready for merging, or does it need work?
I already ran the changes I made with `--check` mode but better safe than sorry.

### CODE REVIEW

File: roles/i18n/tasks/Archlinux.yml
Comment (dianaw353) [inline roles/i18n/tasks/Archlinux.yml:1]: dosent appear to have fonts for korean or japanese. Also id like to avoid closed source fonts when possible

Comment (fkjbvfdvbnfdkjvbjfhd) [reply roles/i18n/tasks/Archlinux.yml:1]: CJK is a term for Chinese, Japanese and Korean all combined. Noto font contains all three of them in that package.

EDIT: As per "closed source fonts," which font are you talking about?

File: roles/hyprland/tasks/Archlinux.yml
Comment (dianaw353) [inline roles/hyprland/tasks/Archlinux.yml:1]: remove hyprshade package plz

File: roles/i18n/tasks/Archlinux.yml
Comment (dianaw353) [reply roles/i18n/tasks/Archlinux.yml:1]: for japan maybe this
[otf-ipafont](https://archlinux.org/packages/?name=otf-ipafont)

File: roles/hyprland/tasks/Archlinux.yml
Comment (fkjbvfdvbnfdkjvbjfhd) [reply roles/hyprland/tasks/Archlinux.yml:1]: Alrighty, right away!

Comment (dianaw353) [reply roles/hyprland/tasks/Archlinux.yml:1]: oh also the ags hyprshade widget i was trying to implement a while ago

Comment (fkjbvfdvbnfdkjvbjfhd) [reply roles/hyprland/tasks/Archlinux.yml:1]: > oh also the ags hyprshade widget i was trying to implement a while ago

Got it!

File: roles/hyprland/files/hypr/config/keybindings/default.conf
Comment (dianaw353) [inline roles/hyprland/files/hypr/config/keybindings/default.conf:1]: can we add these to the ags data shortcuts? I forgot to add em a while back. They are there for keybind passthough for VM

File: roles/i18n/tasks/Archlinux.yml
Comment (fkjbvfdvbnfdkjvbjfhd) [reply roles/i18n/tasks/Archlinux.yml:1]: Alrighty, that might help render some of the characters Noto-CJK might be lacking as well.

File: roles/hyprland/files/hypr/config/keybindings/default.conf
Comment (fkjbvfdvbnfdkjvbjfhd) [reply roles/hyprland/files/hypr/config/keybindings/default.conf:1]: Sure thing, getting on them right now!

**Comment by dianaw353** [comment]
closes #46 #22 (but with hyprsunset instead) #48

**Comment by dianaw353** [comment]
I know this is unrelated but id like to close #11 but would like to add lib32-gnutls as a dependency as ubisoft connect needs it to login... ubisoft connect in our case would require lutris and install it via that and we will have the user install that one manually unless we know of a sulution

**Comment by kurtbahartr** [comment]
> closes #46 #22 (but with hyprsunset instead) #48

Closes: #46 #22 #48 

You don't have to explicitly mention 48 as I already mentioned it in one of the commits before.

**Comment by kurtbahartr** [comment]
> I know this is unrelated but id like to close #11 but would like to add lib32-gnutls as a dependency as ubisoft connect needs it to login... ubisoft connect in our case would require lutris and install it via that and we will have the user install that one manually unless we know of a sulution

Can't we just install both `lib32-gnutls` and `lutris` themselves right away? Is there an additional configuration to be done?

**Comment by dianaw353** [comment]
> > I know this is unrelated but id like to close #11 but would like to add lib32-gnutls as a dependency as ubisoft connect needs it to login... ubisoft connect in our case would require lutris and install it via that and we will have the user install that one manually unless we know of a sulution
> 
> Can't we just install both `lib32-gnutls` and `lutris` themselves right away? Is there an additional configuration to be done?

Sure. Plz add `lib32-gnutls` to the compatibility_layer

**Comment by kurtbahartr** [comment]
Closes: #11

**Pull request merged**